### DISCUSSION TITLE
New project; use Diamond 3.x or wait for 4.x?

### DISCUSSION THREAD

**Discussion by aaronAtVad** [primefaces/community#2503] [PrimeVue Templates] [answered]
Howdy,

Our team is starting a new project using Vue and PrimveVue. This is our first project using Vue and PrimveVue. The project is in the early stages, and expected to ship within 9-12 months.

The Diamond PrimeView template is very close to the page structure and our corporate app theme. Using it would save us a considerable amount of development effort.

We've started the project with primevue@3.50.0, but primevue@4.x was recently release. From my understanding, the Diamond template hasn't been converted to primevue@4.x, but it's on the roadmap for AUG-2024 or Q3-2024.

Should we stick with the primevue@3.x? or move to primevue@4.x?

I don't want to spend a lot of time doing UI/UX/CSS on the 3.x branch if the Diamond template support for 4.x is imminent.

However, I don't want to delay our project waiting for Diamond 4.x if it is delayed past SEPT-2024 or Q3-2024.

What is PrimeTek's track record for delivering according to the road map?

**Comment by tugcekucukoglu** [comment, marked as answer]
Hi,

We are planning to complete Diamond by early September. I can say that we will adhere to the roadmap for template updates.

But if you don't want to wait, you can take Sakai free template as an example. Also, https://www.youtube.com/watch?v=Ni_Yecfxv_g will be helpful.

**Reply by Jmagnowski** [reply]
Hi, 

I have similar problem but with Apollo template. Could you tell me when are you planning to migrate it to version 4.x ?

**Comment by aaronAtVad** [comment]
I just downloaded diamond@6.0.0 which uses primevue@4.x. Will give it a spin. Thanks @tugcekucukoglu !
